taui/src/components/form.js

```diff
@@ -48,10 +48,13 @@ class Form extends React.PureComponent<Props> {
 
     const useNetworks = this.getProfileNetworks(networks, userProfile)
 
+    // TODO: Add networkKey for react-18next's use to the networks as they
+    // come into the app in store.yml and config.json, rather than adding this
+    // 3rd networkName value here
```

Comment:
When adding a TODO / issue pair like this we should also include a link to the tracking issue so that it's clear where to go if someone bumps into this in the code.